src/components/NextScheduleItem/NextScheduleItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import { ScheduleIteType } from '../../pages/BarberHomePage/BarberHomePage';
+import {
+  ClientAndServiceContainer,
+  ClientNameText,
+  Container,
+  ScheduleDateContianer,
+  ServiceTypeText,
+} from './styles';
+
+export default function NextScheduleItem({
+  scheduleItem,
+}: {
+  scheduleItem: ScheduleIteType;
```

Comment:
Onde estão as tipagens?

Reply:
`export interface ScheduleItemType {
  day: number;
  mounth: number;
  year: number;
  from: number;
  to: number;
  name: string;
  whatsapp: string;
  service: string;
}`
Esse é a tipagem, não sei o motivo do git não mostrar